MapzenSDK/Marker.swift

```diff
@@ -411,15 +457,23 @@ public class SystemPointMarker : Marker, GenericSystemPointMarker {
    - parameter coordinates: Coordinates to animate the marker to
    - parameter seconds: Duration in seconds of the animation.
    - parameter ease: Easing to use for animation.
+   - returns: True if successful, false if error or marker has not been added to the map
    */
   public func setPointEased(_ coordinates: TGGeoPoint, seconds: Float, easeType ease: TGEaseType) -> Bool {
-    return tgMarker.setPointEased(coordinates, seconds: seconds, easeType: ease)
+    guard let marker = tgMarker else { return false }
+    marker.pointEased(coordinates, seconds: seconds, easeType: ease)
+    do {
+      try marker.getError()
```

Comment:
The error is only checked on `setPointEased`, I assume that the inconvenience of Swift regarding properties that throw is a potential blocker, but the implementation might leave error incoming from previous usage of the marker, which would be misleading when using `setPointEased`. What seems correct is to check on every modification of the marker or check nowhere at all (while potentially wrapping `getError` to allow its usage externally).